core/refinerycms-core.gemspec

```diff
@@ -27,8 +27,9 @@ Gem::Specification.new do |s|
   s.add_dependency 'railties',                    rails_version
   s.add_dependency 'activerecord',                rails_version
   s.add_dependency 'actionpack',                  rails_version
+  s.add_dependency 'actionmailer',                rails_version
```

Comment:
It was required by Rails 5.0.0.beta3, let me check if it's still relevant: https://github.com/refinery/refinerycms/commit/9219ecf5d56f3a2e8b08ec3dfe8528e9d92f30ca